packages/web-components/src/components/va-radio-option/va-radio-option.tsx

```diff
@@ -56,19 +56,24 @@ export class VaRadioOption {
 
   render() {
     const { checked, ariaDescribedby, name, value, label } = this;
+    const id = this.el.id || (name + value);
     return (
       <Host
         aria-checked={checked ? `${checked}` : 'false'}
         aria-describedby={(checked && ariaDescribedby) || null}
-        aria-label={name}
         checked={checked}
         name={name}
         onClick={() => this.handleChange()}
         role="radio"
         value={value}
-        id={this.el.id || label}
+        id={id}
       >
-        <label htmlFor={label}>{label}</label>
+        <div class="tile">
+          <label htmlFor={id} aria-describedby="description">{label}</label>
+          <div id="description">
+            <slot></slot>
```

Comment:
If we just want the "description" to be text, then I think we should make it be a prop instead of a slot.

Reply:
The "hint" is added below the `va-radio` legend whereas the "description" is added inside the option text.

As for the description, I wasn't sure if there'd ever be a use case for adding a web component, like a `va-additional-info` inside an option. If you don't think it's necessary, then I can change this to a prop.

Reply:
My original question was about adding hint text to the radio **group**. In Matt's response, he said "I don't see why not", but then shared a USWDS screenshot that included the tile variation AND description.

Reply:
```
Group Label
Group Hint text

⚪ Option one
   Description for option one
⚪ Option two
   Description for option two
```

Reply:
I think there was some miscommunication where we thought that the tile + description would solve your problem. If that's not the case, can you remove those changes so they can be added later? I think having a smaller PR would help us understand exactly what is being changed and why.